src/repos/NotificationRepo.ts

```diff
@@ -137,6 +138,27 @@ const notifyNewFlyers = async (flyerIDs: string[]): Promise<void> => {
   });
 };
 
+/**
+ * Send notifications for new flyers, edited flyers, or deleted flyers by organizations followed by the user.
+ */
```

Comment:
I would document the parameters with @param. I know we didn't do it before but it's generally good practice.